hokusai/config.yml

```diff
@@ -1,3 +1,4 @@
----
 project-name: positron
+pre-build: export COMMIT_HASH=$(git rev-parse --short HEAD)
```

Comment:
Can do this in Force too @saolsen @alloy @damassi

Reply:
Ah @eessex with this we can get rid of other hardcoded `COMMIT_HASH=$(git rev-parse HEAD)` scattered about such as [here](https://github.com/artsy/positron/blob/master/.circleci/config.yml#L12) as this hook will ensure it's set before the build phase

Reply:
In force we no longer need this because we removed bucket-assets in favor of strictly Webpack hashing. That work should also eventually be applied over here, but is less urgent since we don't (necessarily) need to worry about long-term caching due to Positron being an internal app.

Reply:
But yah this looks way cleaner than how it was setup in force!

Reply:
Ah that's right!  Thanks @damassi